setup.cfg

```
@@ -0,0 +1,40 @@
+[metadata]
+name = epcsunspecdemo
+version = attr: epcsunspecdemo.__version__
+description = EPC SunSpec demonstration tool
+author = EPC Power Corporation
+url = https://epcpower.com/
+project_urls =
+    Source=https://github.com/epcpower/sunspec-demo
+license = MIT
+classifiers =
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.9
+long_description: file: README.rst
```

So yeah...  I'm confused.  This does actually work as is.  I even tried changing it to `long_description: abcfile: README.rst` and built a wheel and checked and sure enough, the readme wasn't there anymore and the text `abcfile: README.rst` was.  So I guess `configparser` is ok with `:`s instead of `=`s...  who'da thunk.  Though I guess it is \_right\_ there.

https://docs.python.org/3/library/configparser.html#supported-ini-file-structure
> followed by key/value entries separated by a specific string (= or : by default 1)

Anyways, I hate to nitpick but it seems worth being consistent.  `:|`  But I'll but in and just apply this myself.

```suggestion
long_description = file: README.rst
```

---

Consistency is important. I can see why I did it. The previous lines have all those colons and the colon after the `file:`. And then it actually worked. But still... sloppy.